packages/subgraph-service/contracts/utilities/AllocationManagerStorage.sol

```diff
@@ -15,14 +15,14 @@ abstract contract AllocationManagerV1Storage {
     mapping(address indexer => uint256 tokens) public allocationProvisionTracker;
 
     /// @notice Maximum amount of time, in seconds, allowed between presenting POIs to qualify for indexing rewards
-    uint256 public maxPOIStaleness;
+    uint256 internal maxPOIStaleness;
```

Comment:
this could be made public if we make `legacyAllocations` internal? Also i dont think this value is accesible otherwise no?